### PR TITLE
Fix az cli usage docs

### PR DESCRIPTION
**Pull request by colemickens**

### CODE REVIEW

**Comment by colemickens** [comment]
Equal signs were removed because of this bug: https://github.com/Azure/azure-cli/issues/1379 (and then on the other flags just to be consistent.

File: README.md [hunk 1]
```diff
@@ -90,14 +90,14 @@ $ az login
 $ az account set --name "<SUBSCRIPTION NAME OR ID>"
 
 $ az group create \
```
Comment (kmala) [inline README.md]: this should be `az resource group create` instead of `az group create`

Comment (kmala) [inline README.md]: I thought of making a PR but since you already have a PR for the docs change i felt this change also can go along with other changes.

Comment (colemickens) [inline README.md]: Thanks, happy to roll it into this one. Good catch.

**Pull request merged**